package test_4;

import java.sql.Connection;
import java.sql.SQLException;

import com.mysql.jdbc.exceptions.jdbc4.MySQLDataException;

public class Driver {
	static String url = "jdbc:mysql://localhost:3306/test";
	static String user = "root";
	static String password = "541548";
	
	public static void main(String[] args) {
		try {
			Connection myConnect = 
		} catch (SQLException e) {
			e.printStackTrace();
		}
	}
	
}
